src/main.js

```diff
@@ -56,6 +56,12 @@ module.exports = {
       description: 'Paths of node_modules, .eslintignore and others are cached',
       type: 'boolean',
       default: false
+    },
+    fixOnSave: {
+      title: 'Fix errors on save',
+      description: 'Everytime the file is saved, linter-eslint tries to fix errors',
```

Comment:
Maybe change this to something like: "Have eslint attempt to fix some errors automatically when saving the file."?